fwm-docs-qs/azuredeploy.json

```diff
@@ -0,0 +1,464 @@
+                "networkSecurityGroup": {
+                    "id": "[resourceId('Microsoft.Network/networkSecurityGroups', 'nsg-workload-srv')]"
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Network/networkInterfaces",
+            "apiVersion": "2020-03-01",
+            "name": "netInterface-jump-srv",
+            "location": "[resourceGroup().location]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/publicIPAddresses', 'publicIP-jump-srv')]",
+                "[resourceId('Microsoft.Network/virtualNetworks/subnets', 'Spoke-01','Jump-SN')]",
+                "[resourceId('Microsoft.Network/networkSecurityGroups', 'nsg-jump-srv')]"
+            ],
+            "properties": {
+                "ipConfigurations": [
+                    {
+                        "name": "ipconfig1",
+                        "properties": {
+                            "privateIPAllocationMethod": "Dynamic",
+                            "publicIPAddress": {
+                                "id": "[resourceId('Microsoft.Network/publicIPAddresses', 'publicIP-jump-srv')]"
+                            },
+                            "subnet": {
+                                "id": "[resourceId('Microsoft.Network/virtualNetworks/subnets', 'Spoke-01', 'Jump-SN')]"
+                            },
+                            "primary": true,
+                            "privateIPAddressVersion": "IPv4"
+                        }
+                    }
+                ],
+                "enableAcceleratedNetworking": false,
+                "enableIPForwarding": false,
+                "networkSecurityGroup": {
+                    "id": "[resourceId('Microsoft.Network/networkSecurityGroups', 'nsg-jump-srv')]"
+                }
+            }
+        },
+       {
+           "type": "Microsoft.Network/networkSecurityGroups",
+           "apiVersion": "2020-03-01",
+           "name": "nsg-jump-srv",
+           "location": "[resourceGroup().location]",
+           "properties": {
+               "securityRules": [
+                   {
+                       "name": "RDP",
+                       "properties": {
+                           "protocol": "TCP",
+                           "sourcePortRange": "*",
+                           "destinationPortRange": "3389",
+                           "sourceAddressPrefix": "*",
+                           "destinationAddressPrefix": "*",
+                           "access": "Allow",
+                           "priority": 300,
+                           "direction": "Inbound"
+                       }
+                   }
+               ]
+           }
+       },
+       {
+            "type": "Microsoft.Network/networkSecurityGroups",
+            "apiVersion": "2020-03-01",
+            "name": "nsg-workload-srv",
+            "location": "[resourceGroup().location]",
+            "properties": {
```

Comment:
add security rules

Reply:
There are no specific security rules for this VM. It has no public IP address and no specific open ports.

Reply:
why do we have an NSG then?